Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -578,12 +579,17 @@ function New-LDAPSIdentitySource {
     if (($PrimaryUrl -match '^(ldaps:).+((:389)|(:3268))$')) {
         Write-Warning "PrimaryUrl $PrimaryUrl is nonstandard. Are you sure you meant to use the 389/3268 port and not the standard ports for LDAPS, 636 or 3269? Continuing anyway.."
     }
+    Write-Host "Connectivity to $PrimaryUrl is verified."
```

Comment:
Here we just confirmed the format, not the connectivity, I don't think this is worthwhile reporting.

Reply:
Lines 287 and 295 is where I'd expect to report the successful checks.